django_mongodb/operations.py

```diff
@@ -133,3 +134,43 @@ def _prep_lookup_value(self, value, field, field_kind, lookup):
         if field_kind == "DecimalField":
             value = self.adapt_decimalfield_value(value, field.max_digits, field.decimal_places)
         return value
+
+    """Django uses these methods to generate SQL queries before it generates MQL queries."""
```

Comment:
Do we make these SQL queries just for logging/debugging purposes? Could we get away with not rendering them at all?

Reply:
No, we cannot avoid it. It has to do with how this library hooks in to the query generation process. See also:

https://github.com/mongodb-labs/django-mongodb/blob/3c3ad0eb45a2572453aa228ddafac96391dc1ab8/django_mongodb/query.py#L23-L32